frontend/src/features/projects/common/components/PropertyListViewSelect.tsx

```diff
@@ -155,14 +159,15 @@ export const PropertyListViewSelect: React.FC<InputProps> = ({
             name="SelectPropertiesTable"
             columns={columns}
             data={data}
-            lockPageSize
+            dropUp
```

Comment:
What's a `dropUp`?

Reply:
Just has the menu options grow upwards rather than a 'drop-down menu'. Playing off of Goeme's existing functionality. Helps prevent the menu from being cut off / and or the container unnecessarily growing

Reply:
Going to rename this prop so it fits the description better